webapp/src/App.tsx

```diff
@@ -2,7 +2,7 @@ import { Component } from "react"
 import "@patternfly/patternfly/patternfly.css" //have to use this import to customize scss-variables.scss
 
 import { Nav, NavItem, NavList, Page, PageHeader, PageHeaderTools } from "@patternfly/react-core"
-import { ConnectedRouter } from "connected-react-router"
+import { ReduxRouter } from '@lagunovsky/redux-react-router'
```

Comment:
with this dependency defined in `package.json` you should just be able to import ` from 'redux-react-router'` here

Reply:
It's not allow to import like what you suggested.

Reply:
@johnaohara I don't know why CI fail.